modules/metering-debugging.adoc

```diff
@@ -3,8 +3,8 @@
 // * metering/metering-troubleshooting-debugging.adoc
 
 [id="metering-debugging_{context}"]
-
 = Debugging metering
+
 Debugging metering is much easier when you interact directly with the various components. The sections below detail how you can connect and query Presto and Hive, as well as view the dashboards of the Presto and HDFS components.
```

Comment:
no comma

Reply:
I'm always a fan of fewer commas. :clinking_glasses: